src/routes/opening/utils/getSafeAddressFromLogs.ts

```diff
@@ -18,5 +19,5 @@ export const getNewSafeAddressFromLogs = (logs: Log[]): string => {
   // We extract the proxy creation information from the event parameters
   const proxyInformation = proxyCreationEvent.events.find(({ name }) => name === 'proxy')
```

Comment:
Here we don't check that `proxyInformation` is not undefined, and it works because `abi-decoder` doesn't include types. Is it something intentional or just overlooked?

Reply:
We should definitely guard it.